### DISCUSSION TITLE
Github Codespaces unnecessary billing

### DISCUSSION THREAD

**Discussion by wraith4081** [community/community#60086] [Codespaces] [answered]
### Select Topic Area

Question

### Body

Recently, when my Codespaces limit was full, I made an extra addition. Then, after I finished my work, I stopped them all and went to bed. 2 messages came after I went to bed at night. In the first message, he said that 75% was used. In the second message, which came 57 minutes later, it stated that 100% was used and my limit was full. [According to my knowledge](https://github.com/orgs/community/discussions/39697), billing is made only according to the active hour, but it still continues to bill me even though I have no active codespaces. Is this an error?

**Comment by tedster0629** [comment, marked as answer]
Based on your description, it seems like there might be an error or a delay in the billing system. Normally, you should only be billed for the active hours of your Codespaces. If there are no active Codespaces running, you shouldn't be billed for the time when they are stopped.
To resolve this issue, I recommend reaching out to the support team or customer service of the platform or service you are using for your Codespaces. They will be able to investigate the billing discrepancy and provide you with more information regarding the charges. Remember to provide them with specific details, such as the timestamps of the messages and any relevant account information, to help them better understand and resolve the issue.

**Reply by tedster0629** [reply]
Thank you.
